src/main/java/spoon/reflect/declaration/CtTypeInformation.java

```diff
@@ -122,7 +122,12 @@
 
 	/**
 	 * Gets the executables declared by this type and by all its supertypes if
-	 * applicable.
+	 * applicable. This method returns:
+	 * <ul>
+	 *     <li>static, instance and default executables</li>
+	 *     <li>Overridden methods</li>
```

Comment:
Yes, see the test case: https://github.com/INRIA/spoon/pull/655/files#diff-1d756de7906a25260ce271613f43b677R114